packages/client/src/components/widgets/departures/Field.tsx

```diff
@@ -13,14 +13,15 @@ interface FieldProps {
 
 export function Field({ departure }: FieldProps) {
   const { tile } = useColorsOfScheme();
+  console.log(departure.alert);
```

Comment:
Ezt itt le kellene szedni (note to self: eslint szabály erre)

Reply:
sorry, fixáltam